src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -22,6 +22,7 @@
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\ComAwareEventInfo.cs" />
+    <Compile Include="System\Runtime\InteropServices\ComEventInterfaceAttribute.cs" Condition="'$(TargetGroup)' != 'uapaot'" />
```

Comment:
It is being excluded for uapaot builds because it already exists in System.Private.Interop.